webpack.config.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     },
 
     // Enable sourcemaps for debugging webpack's output.
-    devtool: "source-map",
+    devtool: "eval-cheap-source-map",
```

Comment:
Did this just sneak in or was it intended for this PR?

Reply:
My sense is that the current best practice is to use `eval-cheap-module-source-map`  for development and `source-map` for production.